pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -395,8 +395,9 @@ private void checkDeadlockedThreads() {
             String threadNames = Arrays.stream(threadInfos)
                     .map(threadInfo -> threadInfo.getThreadName() + "(tid=" + threadInfo.getThreadId() + ")").collect(
                             Collectors.joining(", "));
-            if (System.currentTimeMillis() - threadDumpLoggedTimestamp
-                    > LOG_THREADDUMP_INTERVAL_WHEN_DEADLOCK_DETECTED) {
+            if ((System.currentTimeMillis() - threadDumpLoggedTimestamp
+                    > LOG_THREADDUMP_INTERVAL_WHEN_DEADLOCK_DETECTED) ||
+                    threadDumpLoggedTimestamp == 0) {
```

Comment:
The condition `threadDumpLoggedTimestamp == 0` is meaningless, right?

Reply:
> The condition `threadDumpLoggedTimestamp == 0` is meaningless, right?

You're right, I'll remove this condition ：`threadDumpLoggedTimestamp == 0`